script/travis_build.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/bash -e
 
-cd "$1" || exit 1
-BUILD_NAME=$1
+BUILD_LOG="$(pwd)/travis.log"
 
-echo "Building $BUILD_NAME"
-
-BUILD_NAME=$(sed 's/[\/+]/_/g' <<< $BUILD_NAME)
-BUILD_NAME=$(sed 's/[\.\-]//g' <<< $BUILD_NAME)
-
-make >> "log_$BUILD_NAME.log"
+cd "$1" > ${BUILD_LOG} || exit 1
+make >> ${BUILD_LOG} || exit 1
+(ctest || ctest -V --rerun-failed) >> ${BUILD_LOG} 
```

Comment:
```suggestion
{
    cd "$1" || exit 1
    make || exit 1
    (ctest || ctest -V --rerun-failed) 
} > "$BUILD_LOG"
```

You can change the last line to
```diff
-} > "$BUILD_LOG"
+} | tee "$BUILD_LOG"
```
if you also want to see it in console instead of tailing it after, but it's probably fine to do as you currently have since it would produce too much output

Reply:
Don't want to see it in console, as exceeded the maximum log length, then build is being terminated

Reply:
I see, then ignore the last half with tee